src/components/Person/Person.jsx

```diff
@@ -1 +1,33 @@
-// export const Person = ({ person }) => ();
+import React from 'react';
+import './Person.scss';
+
+export function Person(person) {
+  const { name, age, sex, isMarried, partnerName } = person.person;
+  const partnerText = sex === 'm'
+    ? `${partnerName} is my wife`
+    : `${partnerName} is my husband`;
+
+  return (
+    <section className="Person">
+      <h2 className="Person__name">
+        {`My name is ${name}`}
+      </h2>
+
+      {
+        age && (
+          <p className="Person__age">
+            {`I am ${age}`}
+          </p>
+        )
+      }
```

Comment:
```suggestion
      {age && (
        <p className="Person__age">
          {`I am ${age}`}
        </p>
      )}
```

fix it everywhere else